pages/common/stressapptest.md

```diff
@@ -0,0 +1,16 @@
+# stressapptest
+
+> Userspace memory and IO test.
+> More information: <https://github.com/stressapptest/stressapptest/>.
+
+- Test the given amount of memory (in Megabytes):
+
+`stressapptest -M {{memory}}`
+
+- Test memory as well as I/O to the given file:
```

Comment:
Maybe we can update it to

```suggestion
- Test memory as well as I/O for the given file:
```

The docs specifies this for the command: `-f filename : add a disk thread with tempfile 'filename' (none)`, so maybe we could update the description to reflect the same.